### DISCUSSION TITLE
Renew Rustdesk Let's Encrypt certificate after expiring

### DISCUSSION THREAD

**Discussion by Affection4036** [rustdesk/rustdesk#9661] [Q&A]
Hello,

What's the easiest way to renew a Rustdesk (Pro) certificate after it has been expired? I thought it would be renewed manually and didn't notice the expiry mails. Would love to do it in a way that has no impact on machines connected to my Rustdesk Pro server. Also would like to know how to prevent this in the future so it will be automatically renewed.

Thank you in advance!

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/license/#renewupgrade-license

**Reply by Affection4036** [reply]
I'm not sure on how renewing my license would work to renew my Let's encrypt certificate? I have about 9 months left on my current license.

**Comment by Affection4036** [comment]
Does anyone know how to replace the Let's Encrypt certificate without creating problems when connecting to PC's in my address book?

**Comment by Axylo** [comment]
try may be this:
`sudo certbot renew`

**Reply by Axylo** [reply]
Oh, I just learnt that disabling IPv6 helps a lot. My web server had both active (IP v4 and IP v6 on public DNS). Once, I put the IP v6 record on hold, it was working.
